src/jit/inline.cpp

```diff
@@ -520,6 +529,51 @@ void InlineContext::Dump(Compiler* compiler, int indent)
     }
 }
 
+//------------------------------------------------------------------------
+// DumpData: Dump a successful InlineContext entry, detailed data, and
+//  any successful descentant inlines
+//
+// Arguments:
+//    compiler - compiler instance doing inlining
+//    indent   - indentation level for this node
+
+void InlineContext::DumpData(Compiler* compiler, int indent)
```

Comment:
I was originally using this to dump the fully detailed tree. As you note, I changed directions and don't use it currently. I'll probably leave it around since it might come in handy later.